runHarness/DataManagers/AuctionKubernetesDataManager.pm

```diff
@@ -298,24 +298,30 @@ sub loadData {
 	$logger->debug("Exec-ing perl /loadData.pl");
 	print $applog "Exec-ing perl /loadData.pl\n";
 	my $kubernetesConfigFile = $cluster->getParamValue('kubeconfigFile');
+	my $context = $cluster->getParamValue('kubeconfigContext');
+	my $contextString = "";
+	if ($context) {
+	  $contextString = "--context=$context";	
+	}
+
 	# Get the list of pods
 	my $cmd;
 	my $outString;	
-	$cmd = "KUBECONFIG=$kubernetesConfigFile kubectl get pod -o=jsonpath='{.items[*].metadata.name}' --selector=impl=auctiondatamanager --namespace=$namespace 2>&1";
+	$cmd = "kubectl get pod -o=jsonpath='{.items[*].metadata.name}' --selector=impl=auctiondatamanager --namespace=$namespace --kubeconfig=$kubernetesConfigFile $contextString 2>&1";
 	$outString = `$cmd`;
 	$logger->debug("Command: $cmd");
 	$logger->debug("Output: $outString");
 	print $applog "Command: $cmd\n";
 	print $applog "Output: $outString\n";
 	my @lines = split /\s+/, $outString;
 	if ($#lines < 0) {
-		$console_logger->error("loadData: There are no pods with label auctiondatamanager in namespace $namespace");
-		exit(-1);
+		$console_logger->error("Data loading failed for Workload $workloadNum, appInstance $appInstanceNum: There are no pods with label auctiondatamanager in namespace $namespace");
+		return 0;
```

Comment:
This is a big change to the error path.  Do tests show that it cleanly exits wv when this occurs?

Reply:
It is actually a bug fix, as the exit(-1) here resulted in Weathervane proceeding with the run despite the inability to load the data.  But your point is good that I need to make sure that this is the proper fix.  I will test that it exits properly when hitting this error.

Reply:
I tested this by killing the datamanager pod between the time it determined that the data wasn't loaded and the attempt to load the data.  The run exited cleanly with the appropriate error messages.